.github/workflows/devToRelease.yaml

```diff
@@ -0,0 +1,39 @@
+name: CI to Release
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!main'
```

Comment:
on:
  push:
    branches:
      - '*'
      - '!main'
 
this means all branches except main
it should be for period-1,2,3,4,5